capella_console_client/tasking.py

```diff
@@ -1,13 +1,133 @@
-from typing import Optional, Dict, Any, List
-from datetime import datetime
+from typing import Optional, Dict, Any, List, Union
```

Comment:
Proposal to rename `tasking.py` to `tasking_request.py` and to make `repeat_requests.py` singular. What do you think?

Reply:
Done, good idea to keep it consistent.